CI / none_ubuntu(16|18)_04 broken: crictl not found in system path

Seen att: https://pipelines.actions.githubusercontent.com/ZGpgH9zUuTeRfFbWuBd3XFWGXuWbsKLh0sZK7k8Ux0yprYkJ4U/_apis/pipelines/1/runs/1256/signedlogcontent/31?urlExpires=2020-03-26T22%3A40%3A11.6985455Z&urlSigningMethod=HMACV1&urlSignature=seWqpr5KzSeF4WVAn950lNBRq3XclREhUZN46NjnjzE%3D

```
 --- FAIL: TestStartStop/group/old-docker (3.46s)
...
2020-03-26T22:11:08.1782700Z         	[preflight] Some fatal errors occurred:
2020-03-26T22:11:08.1783269Z         		[ERROR FileExisting-crictl]: crictl not found in system path
```
```
--- FAIL: TestVersionUpgrade (9.02s)
2020-03-26T22:11:12.8273497Z         		[ERROR FileExisting-crictl]: crictl not found in system path
```

Related: #6799

--- a/.github/workflows/main.yml
+++ b/.github/workflows/main.yml
@@ -274,6 +274,9 @@ jobs:
         sudo apt-get update -qq
         sudo apt-get -qq -y install conntrack
         sudo apt-get -qq -y install socat
+        VERSION="v1.17.0"
+        curl -L https://github.com/kubernetes-sigs/cri-tools/releases/download/$VERSION/crictl-${VERSION}-linux-amd64.tar.gz --output crictl-${VERSION}-linux-amd64.tar.gz
+        sudo tar zxvf crictl-$VERSION-linux-amd64.tar.gz -C /usr/local/bin
     - name: Install gopogh
       shell: bash
       run: |
@@ -352,6 +355,9 @@ jobs:
         sudo apt-get update -qq
         sudo apt-get -qq -y install conntrack
         sudo apt-get -qq -y install socat
+        VERSION="v1.17.0"
+        curl -L https://github.com/kubernetes-sigs/cri-tools/releases/download/$VERSION/crictl-${VERSION}-linux-amd64.tar.gz --output crictl-${VERSION}-linux-amd64.tar.gz
+        sudo tar zxvf crictl-$VERSION-linux-amd64.tar.gz -C /usr/local/bin
     - name: Install gopogh
       shell: bash
       run: |

